acm/acm-icon.el

```diff
@@ -153,6 +153,16 @@
     ("task" . ("material" "text-recognition" "#e22272"))
     ("typedef" . ("material" "label-outline" "#bf11b6"))
     ("tailwind" . ("material" "creation" "#39a6dd"))
+    ("namespace" . ("material" "pan" "#dc5034"))
+    ("package" . ("material" "package" "#dc5034"))
+    ("string" . ("material" "code-string" "#98c807"))
+    ("number" . ("material" "numeric-1" "#98c807"))
+    ("boolean" . ("material" "star-check" "#98c807"))
+    ("array" . ("material" "code-array" "#0085c3"))
+    ("object" . ("material" "steering" "#0085c3"))
+    ("key" . ("material" "account-key" "#0085c3"))
+    ("null" . ("material" "null" "##98c807"))
+    ("typeparameter" . ("material" "format-list-bulleted-type" "#dc5034"))
```

Comment:
You need M-x acm-icon-fetch-all to fetch svg icon to local, then git push those svg files.